docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,41 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work.
+
+If you ever encounter a link that helped you or that is not
+a part of the document and should be, feel free to suggest it
+by creating an issue in our issue tracker.
+
+Git-Links
+---------
+
+-   `Git Tutorial <https://try.github.io/levels/1/challenges/1>`_
```

Comment:
i feel like these hyperlinks can have their descriptions made better. for example: Git Commits -> Commit message guidelines
Git rebase -> How to rebase